core/mm/core_mmu.c

```diff
@@ -2776,7 +2782,7 @@ void core_mmu_init_phys_mem(void)
 		/* Carve out test SDP memory */
 #ifdef TEE_SDP_TEST_MEM_BASE
 		if (TEE_SDP_TEST_MEM_SIZE) {
-			pa = vaddr_to_phys(TEE_SDP_TEST_MEM_BASE);
+			pa = TEE_SDP_TEST_MEM_BASE;
```

Comment:
This fix rather relates to commit 1c1f8b65b that falsely considered SDP_TEST_MEM could be relocated whereas is it not.